 

## Next-Generation Sequencing (NGS)

### General

- **EMBL Online Tutorial** Functional genomics 
	- [I: Introduction and designing experiments](https://www.ebi.ac.uk/training/online/courses/functional-genomics-i-introduction-and-design/)
	- [II: Common technologies and data analysis methods](https://www.ebi.ac.uk/training/online/courses/functional-genomics-ii-common-technologies-and-data-analysis-methods/)
	- [III: Submitting your data and functional genomics databases](https://www.ebi.ac.uk/training/online/courses/functional-genomics-iii-submitting-data/)


- [**SIB** NGS: Quality control, Alignment, Visualisation](https://sib-swiss.github.io/NGS-introduction-training/) | [GitHub](https://github.com/sib-swiss/NGS-introduction-training)
- [**Harvard Chan Bioinformatics Core** Accessing genomic reference and experimental sequencing data](https://hbctraining.github.io/Accessing_public_genomic_data/) | [GitHub](https://github.com/hbctraining/Accessing_public_genomic_data)
- [**CRUK** Workshop on Functional Genomics - summer school 2020](https://bioinformatics-core-shared-training.github.io/cruk-summer-school-2020/) | [GitHub](https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2020)


### RNA-seq

> For most of the courses in this section, some basic experience of using a UNIX/LINUX command line and some R knowledge are assumed and essential. 

- [**CRUK CI** Introduction to Bulk RNA-seq data analysis](https://bioinformatics-core-shared-training.github.io/Bulk_RNAseq_Course_Apr22/) | [GitHub](https://github.com/bioinformatics-core-shared-training/Bulk_RNAseq_Course_Apr22)

- [**Harvard Chan Bioinformatics Core** Introduction to bulk RNA-seq](https://hbctraining.github.io/Intro-to-rnaseq-hpc-salmon-flipped/schedule/links-to-lessons.html) | [GitHub](https://github.com/hbctraining/Intro-to-rnaseq-hpc-salmon-flipped)


- [**Stefano Mangiola** A Tidy Transcriptomics introduction to RNA sequencing analyses](https://stemangiola.github.io/bioc_2020_tidytranscriptomics/) | [GitHub](https://github.com/stemangiola/bioc_2020_tidytranscriptomics/)

- [**UoM Bioinformatics Core Facilities** Guide to RNA-Seq Analysis](https://ycl6.gitbook.io/guide-to-rna-seq-analysis/)

- [**Rockefeller University Bioinformatics Resource Center** RNAseq with Bioconductor](https://rockefelleruniversity.github.io/RU_RNAseq/index.html)  

- [**bioinformaticsworkbook.org** RNA-Seq data Analysis](https://bioinformaticsworkbook.org/dataAnalysis/RNA-Seq/RNA-SeqIntro/RNAseq-using-a-genome.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)

- [**bioinformaticsworkbook.org** Differential Gene Expression analysis](https://bioinformaticsworkbook.org/dataAnalysis/RNA-Seq/RNA-SeqIntro/Differential-Expression-Analysis.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)

- [**SIB** Introduction to RNA-Seq: From quality control to pathway analysis](https://sib-swiss.github.io/RNAseq-introduction-training/) | [GitHub](https://github.com/sib-swiss/RNAseq-introduction-training)
	> Course designed for using a dedicated computer cluster 

- [**Harvard Chan Bioinformatics Core** Differential Gene Expression Analysis](https://hbctraining.github.io/DGE_workshop_salmon_online/schedule/links-to-lessons.html) | [GitHub](https://github.com/hbctraining/DGE_workshop_salmon_online)


- [**EBI** Bulk RNA-seq tutorial](https://mperalc.gitlab.io/bulk_RNA-seq_workshop_2021/) | [GitHub](https://gitlab.com/mperalc/bulk_RNA-seq_workshop_2021/)

### ChIP-seq

- [**Harvard Chan Bioinformatics Core** Introduction to ChIP-seq using high performance computing](https://hbctraining.github.io/Intro-to-ChIPseq/) | [GitHub](https://github.com/hbctraining/Intro-to-ChIPseq)
- [**CRUK CI** Quantitiative analysis of ChiP-seq, ATAC-seq, and related DNA enrichment assays](https://github.com/bioinformatics-core-shared-training/Quantitative-ChIPseq-Workshop)
- [**UMass** Integrated ChIP-seq Data Analysis Workshop](https://hukai916.github.io/IntegratedChIPseqWorkshop/) | [GitHub](https://github.com/hukai916/IntegratedChIPseqWorkshop/)


### ATAC-seq

- [**UMass MCCB** Best practices for ATAC-seq QC and data analysis](https://haibol2016.github.io/ATACseqQCWorkshop/) | [GitHub](https://github.com/haibol2016/ATACseqQCWorkshop/)


### Single cell

- [**SIB** Single-cell Transcriptomics](https://sib-swiss.github.io/single-cell-training/latest/) | [GitHub](https://github.com/sib-swiss/single-cell-training/)

- [**SIB** Advanced topics in single-cell transcriptomics](https://github.com/fmicompbio/adv_scrnaseq_2020)

- [**SIB NBIS/SciLifeLab** Advanced topics in Single Cell Omics](https://nbisweden.github.io/single-cell_sib_scilifelab_2021/) | [GitHub](https://github.com/NBISweden/single-cell_sib_scilifelab_2021)

- [**MGC/BioSB** Course - Single Cell Analysis](https://github.com/LeidenCBC/MGC-BioSB-SingleCellAnalysis2021)

- [**Cambridge Bioinformatics Training Unit** Analysis of single cell RNA-seq data](https://www.singlecellcourse.org/index.html) | [GitHub](https://github.com/hemberg-lab/scRNA.seq.course)  

- [**University of Cambridge / CRUK** Introduction to single-cell RNA-seq data analysis](https://bioinformatics-core-shared-training.github.io/UnivCambridge_ScRnaSeqIntro_Feb2022/)
 
- [**NBIS** Single cell RNA-seq analysis workshop](https://nbisweden.github.io/workshop-scRNAseq/) | [GitHub](https://github.com/nbisweden/workshop-scRNAseq)

- [**Harvard Chan Bioinformatics Core** Single-cell RNA-seq data analysis workshop](https://github.com/hbctraining/scRNA-seq_online) | [GitHub](https://hbctraining.github.io/scRNA-seq_online/schedule/links-to-lessons.html)

- [**WEHI**  Single cell RNA-seq analysis workshop](
https://yunshun.github.io/SingleCellWorkshop/) | [GitHub](https://github.com/yunshun/SingleCellWorkshop/)
	> Integration analysis of multi-sample single cell RNA-seq experiment


- [**Dana-Farber Cancer Institute** Trajectory inference across conditions: differential expression and differential progression](https://kstreet13.github.io/bioc2020trajectories/) | [GitHub](https://github.com/kstreet13/bioc2020trajectories)

- [**ELIXIR EXCELERATE** Single RNA-seq data analysis with R](https://nbisweden.github.io/excelerate-scRNAseq/) | [GitHub](https://github.com/NBISweden/excelerate-scRNAseq)

- [**EBI** Single cell RNA-seq tutorial](https://mperalc.gitlab.io/scRNA-seq_workshop_2021/) | [GitLab](https://gitlab.com/mperalc/scRNA-seq_workshop_2021)




### Variant analysis

- [**SIB** NGS - Genome variant analysis](https://sib-swiss.github.io/NGS-variants-training/) | [GitHub](https://github.com/sib-swiss/NGS-variants-training/)
- [**bioinformaticsworkbook.org** Variant Discovery](https://bioinformaticsworkbook.org/dataAnalysis/VariantCalling/variant-calling-index.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)

- [**WEHI** A introduction to SNV analyses in whole genome sequencing](https://github.com/PapenfussLab/IntroductionToGenomicsWorkshop/) | [GitHub](https://papenfusslab.github.io/IntroductionToGenomicsWorkshop/)
- [**Waldron lab CUNY** Copy number variation analysis with Bioconductor](https://waldronlab.io/CNVWorkshop/) | [GitHub](https://github.com/waldronlab/CNVWorkshop)


### Genome assembly

- [**bioinformaticsworkbook.org** Introduction to Genome Assembly](https://bioinformaticsworkbook.org/dataAnalysis/GenomeAssembly/Intro_GenomeAssembly.html#gsc.tab=0) | [GitHub](https://github.com/ISUgenomics/bioinformatics-workbook)


### Proteomics
- [**EMBL-EBI** Proteomics bioinformatics](https://www.ebi.ac.uk/training/events/proteomics-bioinformatics-2021/)
	> * Materials and live recording available. 
	> * MaxQuant/Perseus tutorial: Day 3
	> * MsqRob tutorial: Day 4  (Practical on Downstream quantitative analysis/statistics)

### Pathway analysis

- [**EMBL-EBI** Reactome Quick tour](https://www.ebi.ac.uk/training/online/courses/reactome-quick-tour/)

### Miscellaneous

- [**James W. MacDonald** Introduction to Bioconductor annotation resources](https://jmacdon.github.io/Bioc2020Anno/) | [GitHub](https://github.com/jmacdon/Bioc2020Anno)

- [**Waldron lab CUNY** Public data resources and Bioconductor](https://waldronlab.io/PublicDataResources/) | [GitHub](https://github.com/jmacdon/Bioc2020Anno)

- [**Waldron lab CUNY** Functional enrichment analysis of high-throughput omics data](https://waldronlab.io/enrichOmics/) | [GitHub](https://github.com/waldronlab/enrichOmics)

- [**WEHI**  Gene-set enrichment analysis workshop](https://davislaboratory.github.io/GenesetAnalysisWorkflow/) | [GitHub](https://github.com/DavisLaboratory/GenesetAnalysisWorkflow/)

- [**Waldron lab CUNY** Multi-omic Integration of cBioPortal and TCGA data with MultiAssayExperiment](https://waldronlab.io/MultiAssayWorkshop/) | [GitHub](https://github.com/waldronlab/MultiAssayWorkshop/)

- [**SIB** Enrichment Analysis](https://sib-swiss.github.io/enrichment-analysis-training/) | [GitHub](https://github.com/sib-swiss/enrichment-analysis-training/)




